tools/EFToolsVsReferences.settings.targets

```diff
@@ -11,8 +11,10 @@
     <Reference Include="Microsoft.VisualStudio.Shell.Immutable.12.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.Interfaces.14.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.14.0" />
+    <Reference Include="Microsoft.VisualStudio.Telemetry" />
```

Comment:
Should we have an itemgroup for VisualStudioVersion greater than 14? The current references there seem to be non overlapping.